output/json/json.go

```diff
@@ -171,6 +170,29 @@ func (o *Output) handleMetric(m *metrics.Metric, jw *jwriter.Writer) {
 	}
 	o.seenMetrics[m.Name] = struct{}{}
 
-	wrapMetric(m).MarshalEasyJSON(jw)
+	//nolint:exhaustruct
+	wrapped := metricEnvelope{
+		Type:   "Metric",
+		Metric: m.Name,
+	}
+	wrapped.Data.Name = m.Name
+	wrapped.Data.Type = m.Type
+	wrapped.Data.Contains = m.Contains
+
+	if ts, ok := o.thresholds[m.Name]; ok {
+		wrapped.Data.Thresholds = ts
+	}
+
+	// TODO: refactor after the other refactors
+	// in the metrics area will be completed.
+	//
+	// This parts can be racy because
+	// they are controlled and written from the metrics.Engine.
+	wrapped.Data.Tainted = m.Tainted
+	if len(m.Submetrics) > 0 {
+		wrapped.Data.Submetrics = m.Submetrics
+	}
```

Comment:
```suggestion
	wrapped.Data.Submetrics = m.Submetrics
```

I see no need for an extra `if`